src/unix/mod.rs

```diff
@@ -447,6 +458,11 @@ extern {
                link_name = "waitpid$UNIX2003")]
     pub fn waitpid(pid: pid_t, status: *mut ::c_int, options: ::c_int)
                    -> pid_t;
+    #[cfg(not(target_os = "openbsd"))] // " if " -- appease style checker
```

Comment:
Can we defer that conversation until after I have something that _works_ everywhere, please?  I don't think the right tradeoff among tidiness, consistency with other stuff, and minimization of repetition will be apparent till then.